bigbluebutton-html5/private/config/settings.yml

```diff
@@ -320,6 +320,13 @@ public:
     server: { enabled: true, level: info }
     console: { enabled: true, level: debug }
     external: { enabled: false, level: info, url: https://LOG_HOST/html5Log, method: POST, throttleInterval: 400, flushOnClose: true, logTag: "" }
+  fallbackLanguages:
```

Comment:
Can you please move the languages out of settings.yml please? languages-map.yml or json would be more appropriate imo